app/api.js

```diff
@@ -1,19 +1,21 @@
 const SITE_ENDPOINT = '/site';
 
-export function fetchSiteState() {
-  return fetch(SITE_ENDPOINT, {
-    credentials: 'same-origin',
-  })
-  .then(response => {
-    if (response.status === 200) return response.json();
-    throw new Error('Unable to load site content from server');
-  })
-  .catch(error => { throw error; });
+function assertStatusOK(response) {
+  if (response.status >= 200 && response.status < 300) {
+    return response;
+  }
+  const error = new Error(response.statusText);
+  error.response = response;
+  throw error;
 }
 
-export function publishSiteState({ community, conference, events }) {
-  const body = JSON.stringify({ community, conference, events });
-  return fetch(SITE_ENDPOINT, {
+function parseJSON(response) {
+  return response.json();
+}
+
+function post(url, object) {
+  const body = JSON.stringify(object);
+  return fetch(url, {
```

Comment:
Yee. Now that I look we don't actually polyfill `global.fetch`. Should probably do that, even though this site is only used internally.